### PR TITLE
Add Helm integration subproject

### PR DESCRIPTION
**Pull request by johnbelamaric**
cc @henderiw @ciaranjohnston

### CODE REVIEW

**Review by gvbalaji — APPROVED** [review]
/lgtm

**Comment by henderiw** [comment]
I was thinking this morning on this when I woke up. What we agreed to do is gather requirements. Would this not be more appropriate to handle this as part of SIG1 ?

**Comment by johnbelamaric** [comment]
That would be fine too. Or a cross SIG group of some sort. In K8s we have an official concept of a "working group" which is a cross-SIG, temporary group with a specific dissolution criteria. That would be the appropriate type of group for this problem. We can check the doc to see if we have a similar construct. Or just organize it out of SIG 1 for now and collaborate informally.

**Comment by s3wong** [comment]
> I was thinking this morning on this when I woke up. What we agreed to do is gather requirements. Would this not be more appropriate to handle this as part of SIG1 ?

@henderiw I explicitly asked this question on #sig-automation channel on Slack (https://nephio.slack.com/archives/C03MB5GRATS/p1677797179661959)  --- to summarize, I was asking if the main goal of Helm support is for (a) CSPs because they love Helm (?), or (b) ease of NF vendor to onboard Nephio; the answer I got was that it is primarily for (b), if so, then I don't think we need to run this through SIG-1 as this work should NOT be user impacting.

Of course, if the subgroup ends up making a recommendation that Nephio should provide support for use case (a) as well, then the subgroup should draft a proposal and enlist SIG-1 for user requirements. But as it stands now, my understanding is that the primary use case is (b)

File: sig-automation/README.md [hunk 1]
```diff
@@ -174,3 +174,24 @@ In terms of known R1 components, this subproject will focus on:
...
+
+This group will study and propose how Helm integrates with Nephio. The goals
+include:
+- Articulate the goals in supporting Helm. For example, is this to primarily
+  to leverage existing investements, or for some additional reasons.
+- Discover vendor, carrier, and other stakeholder requirements
+- Identify and document existing integration points and best practices
+- Identify Nephio changes that are needed to meet the goals and requirements
+
+**Scope**
```
Comment (ns-sundar) [inline sig-automation/README.md:189]: This is good as it is. Do we want to state the deliverables of this subproject, such as maybe a doc that states the pros/cons of the different options considered and the rationale for the final recommendation?

Comment (johnbelamaric) [reply sig-automation/README.md:189]: See latest commit and resolve the conversation if it's OK.

**Review by ns-sundar — APPROVED** [review]

File: sig-automation/README.md (hunk 1 above)
Comment (ns-sundar) [reply sig-automation/README.md:189]: Thanks, John.

**Comment by johnbelamaric** [comment]
I'll give this one more day for comments then merge it tomorrow.

**Pull request merged**